typescript/create-onchain-agent/src/constants.ts

```diff
@@ -21,7 +21,12 @@ export const EVM_NETWORKS: EVMNetwork[] = [
 
 export const SVM_NETWORKS: SVMNetwork[] = ["solana-mainnet", "solana-devnet", "solana-testnet"];
 
-const CDP_SUPPORTED_EVM_WALLET_PROVIDERS: WalletProviderChoice[] = ["CDP", "Viem", "Privy"];
+const CDP_SUPPORTED_EVM_WALLET_PROVIDERS: WalletProviderChoice[] = [
+  "CDP",
+  "SmartWallet",
```

Comment:
opinion: What should the name be? This determines the display name in the CLI when picking your wallet provider.

Should it be:
* "SmartWallet" (conventionally correct, since all the other options are the prefix of their associated WalletProvider, and the last two-word prefix was SolanaKeypair)
* "Smart Wallet" (more grammatically correct)
* "CDP Smart Wallet" (best branding)